Taller152/src/paqueteuno/Principal.java

```diff
@@ -12,5 +12,20 @@
  * @author reroes
  */
 public class Principal {
+    public static void main(String[] args) {
+        String cadenaFinal = "";
+        String[] paises = {"Ecuador", "Colombia", "Perú", "Venezuela", "Chile",
+            "Bolivia"};
+        String[] capitales = {"Quito", "Bogotá", "Lima", "Caracas", "Santiago",
+            "La Paz"};
+        int[] poblacion = {17643054, 51152371, 33050325, 28435940, 19116201,
+            11673021};
+        for (int i = 0; i < paises.length; i++) {
+            cadenaFinal = String.format("%s%s, con capital %s tiene "
+                    + "una población de %s\n",cadenaFinal,paises[i], 
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.